packages/server/test/unit/browsers/chrome_spec.js

```diff
@@ -117,7 +117,22 @@ describe('lib/browsers/chrome', () => {
     it('sets default window size and DPR in headless mode', function () {
       chrome._writeExtension.restore()
 
-      return chrome.open({ isHeadless: true }, 'http://', openOpts, this.automation)
+      return chrome.open({ isHeadless: true, majorVersion: 112 }, 'http://', openOpts, this.automation)
+      .then(() => {
+        const args = launch.launch.firstCall.args[3]
+
+        expect(args).to.include.members([
+          '--headless=new',
+          '--window-size=1280,720',
+          '--force-device-scale-factor=1',
+        ])
+      })
+    })
+
+    it('sets headless in the old style for versions lower than 112', function () {
```

Comment:
Should we add a test to ensure users can override and set `--headless` for v112+ browser to ensure workaround behaves as expected?

Reply:
I'll manually test this to make sure but i don't think we'd need to write a test for this since it's the same functionality that already exists for overriding chrome flags.

```js
setupNodeEvents: function setupNodeEvents(on, config) {
  on('before:browser:launch', (browser = {}, launchOptions) => {
    if (browser.name === 'chrome' && browser.isHeadless) {
        launchOptions.args.push('--headless')
    }
    return launchOptions;
  });
}
```